crates/autopilot/src/arguments.rs

```diff
@@ -169,11 +169,13 @@ pub struct Arguments {
 
     /// The amount of time that the autopilot waits looking for a settlement
     /// transaction onchain after the driver acknowledges the receipt of a
-    /// settlement.
+    /// settlement. Since the autopilot now concurrently awaits the settlement
+    /// transaction, this timeout is extended to ensure the task can complete
+    /// execution.
     #[clap(
         long,
         env,
-        default_value = "1m",
+        default_value = "5m",
```

Comment:
This value could be increased even further. Failing by timeout means cancellation of the blockchain monitoring task.